### PR TITLE
Add Symfony Uid support

### LINKED ISSUE #3714: Symfony Uid support for Json schema
**Description**  

Since Symfony 5.1 the Uid component was introduced and when relying only on it instead of ramsey/uuid,
wrong schema is generated, because it detects it as an object instead of string:uuid.
There is a pull request to symfony for Uid normalizer https://github.com/symfony/symfony/pull/36406/files,
but there are two missing pieces:
 - symfony/uid identifier normalizer;
 - json schema to support uuid/ulid;

I can open separate PR's for both of these, but we should decide if the second point is a bugfix or a new feature?

**Example**  
none

### PR DESCRIPTION
**Pull request by norkunas**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tickets       | fixes #3714
| License       | MIT
| Doc PR        | api-platform/docs#...

- Added support for denormalizing identifiers with symfony uuid/ulid;
- Added ulid/uuid support to json schema;

### CODE REVIEW

**Review by alanpoulain — APPROVED** [review]
Very nice!

**Comment by alanpoulain** [comment]
Thanks a lot @norkunas!

**Comment by alanpoulain** [comment]
See also: https://github.com/symfony/symfony/pull/36406.

File: src/Bridge/Symfony/Bundle/Resources/config/symfony_uid.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="api_platform.identifier.symfony_ulid_normalizer" class="ApiPlatform\Core\Bridge\Symfony\Identifier\Normalizer\UlidNormalizer" public="false">
```
Comment (maxhelias) [inline src/Bridge/Symfony/Bundle/Resources/config/symfony_uid.xml:8]: Why not make the two Normalizers in one? 
Like this : https://github.com/symfony/symfony/blob/dafcc58e6bd8553a6f461ff585322206ffd81956/src/Symfony/Component/Serializer/Normalizer/UidNormalizer.php#L40-L55

File: src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php [hunk 2]
```diff
@@ -53,6 +53,7 @@
 use Symfony\Component\Finder\Finder;
 use Symfony\Component\HttpClient\HttpClientTrait;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\Component\Uid\AbstractUid;
 use Symfony\Component\Validator\Validator\ValidatorInterface;
```
Comment (Wyze5280) [inline src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php:57]:
```suggestion
 use Symfony\Component\Validator\Validator\ValidatorInterface;
```

File: src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php [hunk 3]
```diff
@@ -53,6 +53,7 @@
...
 use Symfony\Component\Yaml\Yaml;
```
Comment (Wyze5280) [inline src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php:58]:
```suggestion
 use Symfony\Component\Yaml\Yaml;
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -113,6 +113,8 @@ jobs:
         run: |
           composer global require --prefer-dist --no-progress --no-suggest --ansi \
```
Comment (Wyze5280) [inline .github/workflows/ci.yml:114]:
```suggestion
           composer global require --prefer-dist --no-progress --no-suggest --ansi \
```

File: src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php [hunk 5]
```diff
@@ -53,6 +53,7 @@
 use Symfony\Component\Finder\Finder;
```
Comment (Wyze5280) [inline src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php:53]:
```suggestion
 use Symfony\Component\Finder\Finder;
```

**Review by Wyze5280 — APPROVED** [review]

**Pull request merged**